lib/response.js

```diff
@@ -166,12 +166,12 @@ module.exports = {
     }
 
     // stream
-    if ('function' == typeof val.pipe) {
+    if ('function' === typeof val.pipe) {
       onFinish(this.res, destroy.bind(null, val));
       ensureErrorHandler(val, err => this.ctx.onerror(err));
 
       // overwriting
-      if (null != original && original != val) this.remove('Content-Length');
+      if (null != original && original !== val) this.remove('Content-Length');
```

Comment:
```suggestion
      if (null !== original && original !== val) this.remove('Content-Length');
```

Reply:
Given the nature of type coercion with nulls, I'm not sure this will be a safe move.

`null == undefined
true
null === undefined
false`